.github/workflows/clippy.yml

```diff
@@ -19,4 +19,4 @@ jobs:
       env:
         OCAML_VERSION: 4.09.1
         OCAML_WHERE_PATH: /ignored
-      run: cargo +nightly clippy --features=without-ocamlopt --all -- -D warnings
+      run: sudo apt install ocaml && cargo +nightly clippy --features=without-ocamlopt --all -- -D warnings
```

Comment:
What made OCaml a necessity here? I had to install it to be able to compile ocalm-boxroot-sys at first, but after I added a bunch of fake headers and the `without-ocamlopt` option it stoped being a requirement.

Reply:
I'm not sure, but it looks like it is coming from `ocaml-boxroot-sys`
```
warning: In file included from vendor/boxroot/boxroot.c:19:
warning: vendor/boxroot/boxroot.h:4:10: fatal error: caml/mlvalues.h: No such file or directory
warning:     4 | #include <caml/mlvalues.h>
warning:       |          ^~~~~~~~~~~~~~~~~
warning: compilation terminated.
```

Reply:
Oh I see. I "fixed" that by adding some fake headers, but that only works if you are targetting OCaml <= 4.09, of you are not going to link the program (like here or when generating docs in docs.rs).

The issue is that because ocaml-boxroot-sys compiles C code, it really needs the headers to be available.

Reply:
Related to this, I have been thinking that it would be useful if `ocaml-sys` could derive the OCaml version from just the `OCAML_WHERE` path (by looking at the headers).